README.md

```diff
@@ -11,7 +11,10 @@ More specifically, the created site:
 - uses a gem-based approach, i.e. uses a `Gemfile` and loads the `just-the-docs` gem;
 - uses the [GitHub Pages / Actions workflow] to build and publish the site on GitHub Pages.
 
-To get started with creating a site, just click "[use this template]"!
+To get started with creating a site, simply:
+
+1. click "[use this template]" to create a GitHub repository
+2. go to Settings > Pages > Build and deployment > Source, and select GitHub Actions
```

Comment:
This still ends up not building the site _per se_ — after cloning the template (and initial commit/push), workflows are run, and failing. After enabling the pages publishing source, it's either:
1. re-run the failed action to attempt deploying again, or
2. edit content and push a new commit to trigger a build

This will be obvious to anyone using GHA or Pages already, but might trip up someone just beginning…

(Maybe adding a note that only subsequent pushes will be able to deploy from that point on? I'm thinking whether some addition like this wouldn't make sense to propose…)

Reply:
(Ah, nevermind, it's mentioned further down the page — incl. re-running any failed builds etc., guess that's sufficient for anyone a bit lost and following it step by step…)